bullet_train-api/lib/jbuilder/values_transformer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "jbuilder"
+
+module ValuesTransformer
+  def _set_value(key, value)
+    value = value.body if value.is_a?(ActionText::RichText)
```

Comment:
This reminds me of the comment in the Transformer about rich text attributes

https://github.com/bullet-train-co/bullet_train-core/blob/aad074837f28f3dae7351707225bf8d385bf4047/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb#L1140

We skip trix editor attributes when super scaffolding jbuilder files, but should we add the body somehow over there in the transformer as well?

Reply:
Well, anyway those attributes are exposed for `ScaffoldingCompletelyConcreteTangibleThing` when `HIDE_THINGS` is not enabled. So this Jbuilder hack lets us show `ActionText::RichText` attributes properly (as `string` instead of whole object). I think if those were skipped before, now they can be added. Also, if there are any other attributes that should be exposed differently in Jbuilder, they might also be added to this file.

I afraid I didn't fully understand what you meant by `should we add the body somehow over there in the transformer as well?` :)

Reply:
> I afraid I didn't fully understand what you meant by should we add the body somehow over there in the transformer as well? :)

I just wasn't sure how we should handle rich text attributes with the API, but after looking into it a bit, I'm thinking I'll make a different PR to return the `ActionText::RichText` object as a whole instead of just the body (that seems to make the most sense to me). As far as what we return, I think we can leave the decision up to Andrew. I'm fine with how things are now though. Thanks!

Here is an example of `bar` as a trix editor attribute:
![Screenshot from 2023-05-08 20-22-11](https://user-images.githubusercontent.com/10546292/236811640-aa550dc3-e772-46e2-acac-98590b3ea0ee.png)

Reply:
Well, that data in API feels completely useless for me, as it is the internal ActionText stuff. Also, I am pretty sure warnings were there as when we create a record, we send `bar` as a string. And I'm not sure we now support different types of same attribute for Attributes and Parameters. Should we?

Reply:
> Well, that data in API feels completely useless for me, as it is the internal ActionText stuff, and as an API user I can't do anything with that, it's only clutters up the information.

Good point, I didn't consider this. I hadn't run the the ActionText::RichText API object against the OpenAPI test so that makes sense about the warnings. Thanks for explaining!